evap/staff/forms.py

```diff
@@ -702,6 +703,10 @@ def save(self, *args, **kw):
 
         self.instance.is_active = not self.cleaned_data.get('is_inactive')
 
+        # refresh results cache
+        for evaluation in Evaluation.objects.filter(contributions__contributor=self.instance).distinct():
```

Comment:
this should only be done when the name actually changed. when, e.g., just updating the `email`, we don't have to refresh the results.